### DISCUSSION TITLE
Convert from JPEG to JXL, file size become much larger.

### DISCUSSION THREAD

**Discussion by tvcat** [T8RIN/ImageToolbox#967] [Q&A] [answered]
Version: 2.7.0 FOSS
Android: 14 (One UI 6.0)
Original JPEG size: 5.3MB
Resize: no
JXL Lossless: Effort =7, Speed=2
Converted size: 38MB

Compare to when I convert with XL converter (Effort=7 Lossless) )on Windows 10. The converted size is 3.13MB
Is this normal? I'm expecting smaller file size.

**Comment by T8RIN** [comment, marked as answer]
Use jxl tools for this needs, transcoding

**Reply by tvcat** [reply]
I see, thanks.

**Comment by T8RIN** [comment]
It's presented in beta versions
